hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1034,6 +1034,29 @@ public function upgrade_1025() {
   }
 
   /**
+   * Add options to choose bi-monthly or bi-weekly for pay cycle
+   *
+   * @return bool
+   */
+  public function upgrade_1026() {
```

Comment:
Sorry I missed this earlier, but you should call this method  at the end of upgradeBundle() function in this file.  This is so that on new installations your upgrade gets run also and you don't have to worry about making the same change twice (installation vs upgrade scripts). Just call your upgrader function after this line:

https://github.com/civicrm/civihr/blob/d335e1d80c9c68fc5e06a88c37a890d0fcfd1fd1/hrjobcontract/CRM/Hrjobcontract/Upgrader.php#L545

On some other extensions this is done in a little smarter way, so we don't have to maintain this upgradeBundle method by hand, but for now, hrjobcontract still requires this.